ups/meas_base.cfg

```diff
@@ -21,5 +21,5 @@ config = lsst.sconsUtils.Configuration(
     __file__,
     headers=["lsst/meas/base.h"],
     hasDoxygenInclude=False,
-    hasSwigFiles=True,
+    hasSwigFiles=False,
```

Comment:
It shouldn't matter anymore, as this is only used to set the Swig include path for dependent packages.  I suppose the more correct value is `False`, though we could also just change the default (or even remove the option) in the pybind11 branch of sconsUtils.